doc/help/commands.asciidoc

```diff
@@ -1163,6 +1163,7 @@ Spawn a command in a shell.
  - `/usr/share/qutebrowser/userscripts`
 
 * +*-v*+, +*--verbose*+: Show notifications when the command started/exited.
+* +*-v*+, +*--output-to-tab*+: Show stderr, stdout, and exit status in a new tab.
```

Comment:
The `-v` here looks like you've been editing this file directly, don't do that - see the comment at the beginning of the file :wink: